package.json

```diff
@@ -114,7 +114,7 @@
     "color-normalize": "1.5.0",
     "color-parse": "1.3.8",
     "color-rgba": "2.1.1",
-    "electron": "11.2.1",
+    "electron": "6.1.7",
```

Comment:
You'll need to remove this change, as it's Mac specific. There is a way to do this through a git exlude file besides gitignore, but just manually discard the changes for now.